content/en/UI.json

```diff
@@ -1,8 +1,14 @@
 {
   "UI": {
     "Language": "English",
+    "Flag": "🇺🇸",
```

Comment:
I reckon this is a hot take for the flag to associate with English :wink:

Reply:
Indeed